src/TMap.cpp

```diff
@@ -2682,11 +2682,8 @@ void TMap::slot_replyFinished(QNetworkReply* reply)
         qWarning() << "TMap::slot_replyFinished( QNetworkReply * ) ERROR - received argument was not the expected stored pointer.";
     }
 
-    if (reply->error() != QNetworkReply::NoError) {
-        if (reply->error() != QNetworkReply::OperationCanceledError) {
-            // Don't post an error for the cancel case - it has already been done
-            const QString alertMsg = tr("[ ALERT ] - Map download failed, error reported was:\n%1.").arg(reply->errorString());
-            postMessage(alertMsg);
+    if (reply->error() != QNetworkReply::NoError && reply->error() != QNetworkReply::OperationCanceledError) {
+            // Don't report on any errors here as we've already done so in slot_downloadError(...) previously.
             cleanup();
             return;
         }
```

Comment:
With the 2 if statements combined into one opening `{`, the inner closing `}` wasn't taken off.
```suggestion
    if (reply->error() != QNetworkReply::NoError && reply->error() != QNetworkReply::OperationCanceledError) {
        // Don't report on any errors here as we've already done so in slot_downloadError(...) previously.
        cleanup();
        return;
```

Reply:
Shame bugfix, sorry